net-im/profanity/profanity-0.14.0-r1.ebuild

```diff
@@ -0,0 +1,96 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+PYTHON_COMPAT=( python3_{10..11} )
+inherit python-single-r1
+
+DESCRIPTION="A console based XMPP client inspired by Irssi"
+HOMEPAGE="https://profanity-im.github.io"
+SRC_URI="
+    https://github.com/profanity-im/profanity/releases/download/${PV}/${P}.tar.gz
+    https://github.com/profanity-im/profanity/commit/122434a.patch
+        -> ${PN}-ox-carbons.patch
```

Comment:
Please use $PN-0.14.0-foo-bar.patch as naming scheme for the patches.

Reply:
Done